workshops/read_the_internet/README.md

```diff
@@ -0,0 +1,175 @@
+```python
+# get table rows
+tableRows = table.findAll('tr')
+```
+Now that we have all the rows in a Python list, we can iterate through the list and try to find the United States or you can input any Country you want.
+
+Before getting to the looping part though we will need to take a look at some of our outputs. This may happen to you when webscraping, the text has weird formatting. In these next steps I'm going to show the text from the website with the weird formatting and how we can use Python to remove that.
+
+I will just use the second row and first column for this example
+```python
+tableCols = tableRows[1].findAll('td')
+tableCol = tableCols[0]
+tableCol.text
+```
+This returns `'\r\n          TOTAL\r\n        '` with weird spacing and newline and indents. With Python we can use the `strip` function on the string and we will end with this code `tableCol.text.strip()` and this text `'TOTAL'`. Now that we got that handled lets get to reading the data!
+
+First we have a have a for each loop to loop through every row in the table. You make notice the `[1:]`, this was added to make the loop just a little more effcient because the first row at index `[0]` is just the table header with no data. Next we use an if statment to check if the current row's country is the United States (This can be replaced by any country). **Do Note** that we use the `.find()` function because we are only checking the first column and `.text[2:]` because the website has a weird astricks we need to remove. Once we find the country of our choice, we use `findAll()` to get all the columns in the row and print out some data.
+```python
+for row in tableRows[1:]:
+    if 'United States' == row.find('td').text[2:].strip():
+        columns = row.findAll('td')
+        print(columns[0].text[2:].strip())
+        print(f'Confirmed Cases: {columns[1].text.strip()}')
+        print(f'New Cases: {columns[3].text.strip()}')
+        print(f'Critical Cases: {columns[5].text.strip()}')
+```
+The output at the time of writing this workshop is:
+```
+United States
+Confirmed Cases: 9,566,690
+New Cases: 28,845
+Critical Cases: 88,127
+```
+Scary Numbers! Hopefully at the time of reading this workshop things won't be too much worse :(
+
+Here is some example code for checking for multiple countries
+```python
+countries = ['United States', 'Spain', 'Canada']
+for row in tableRows[1:]:
+    if row.find('td').text[2:].strip() in countries:
+        columns = row.findAll('td')
+        print(columns[0].text[2:].strip())
+        print(f'Confirmed Cases: {columns[1].text.strip()}')
+        print(f'New Cases: {columns[3].text.strip()}')
+        print(f'Critical Cases: {columns[5].text.strip()}\n')
+```
+
+### Thats it! You can now get live COVID19 data but more importantly you are now primed with knowledge to webscrape anything!
+![Sponge bob done gif](https://cloud-5bmb4t5tp.vercel.app/0completed.gif)
```

Comment:
Click on "Preview" in this GitHub file and scroll down here. You may notice that the text appears next to the image instead of under it. This is because single new lines in markdown aren't interpreted as new lines. This is a problem I've been noticing throughout the workshop. It looks weird at first, but the proper formatting should be:

```
### That's it! You can now get live COVID19 data but more importantly you are now primed with knowledge to webscrape anything!

![Sponge bob done gif](url)

following paragraph
```

That way it'll look correct on the website